### PR TITLE
Add DDPG example

### PR DESCRIPTION
**Pull request by ManosMagnus**

### CODE REVIEW

File: examples/torch/ddpg_example/controllers/supervisor/.#supervisor_manager.py [hunk 1]
```diff
@@ -0,0 +1 @@
+manos@oikos-station.8431:1576666746
```
Comment (tsampazk) [inline examples/torch/ddpg_example/controllers/supervisor/.#supervisor_manager.py]: This needs to be removed

File: examples/torch/ddpg_example/controllers/utilities.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    # print("x_f: {} , z_f: {}".format(x_f, z_f) )
+    if is_true_angle:
+        x_f = -x_f
+    angleDif = math.atan2(z_f, x_f)
+
+    if is_abs:
+        angleDif = abs(angleDif)
+
+    return angleDif
```
Comment (tsampazk) [inline examples/torch/ddpg_example/controllers/utilities.py]: This file is probably a duplicate of the utilities.py inside the supervisor directory

File: examples/torch/ddpg_example/controllers/supervisor/models/networks.py [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+    def save_checkpoint(self):
+        print("...saving checkpoint....")
+        T.save(self.state_dict(), self.checkpoint_file)
+
+    def load_checkpoint(self):
+        print("..loading checkpoint...")
+        self.load_state_dict(T.load(self.checkpoint_file))
+
+
+class Agent(object):
```
Comment (tsampazk) [inline examples/torch/ddpg_example/controllers/supervisor/models/networks.py]: Agent class is basically the DDPG algorithm. I think it shouldn't be inside the networks.py file

**Comment by ManosMagnus** [comment]
This PR is under construction, it is not ready to be merged

**Pull request closed without merging**